src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -41,6 +41,17 @@ exception. The attribute value will be set to full name of exception type.
   path.
   ([#5044](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5044))
 
+* Fixes the `http.route` attribute for scenarios in which it was
+  previously missing or incorrect. Additionally, the `http.route` attribute
+  is now the same for both the metric and `Activity` emitted for a request.
+  Lastly, the `Activity.DisplayName` has been adjusted to have the format
+  `{http.request.method} {http.route}` to conform with [the specification](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#name).
+  There remain scenarios when using conventional routing or Razor pages where
+  `http.route` is still incorrect. See [#5056](https://github.com/open-telemetry/opentelemetry-dotnet/issues/5056)
+  and [#5057](https://github.com/open-telemetry/opentelemetry-dotnet/issues/5057)
+  for more details.
```

Comment:
Opened these issues to highlight the remaining known issues related to `http.route`.

@JamesNK I'm interested in your thoughts. I'm hoping we can coordinate to resolve these issues.